mlflow/tracking/client.py

```diff
@@ -2667,6 +2671,26 @@ def create_model_version(
             await_creation_for=await_creation_for,
         )
 
+    def copy_model_version(self, src_model_uri, dest_name) -> ModelVersion:
+        """
+        Copy a model version from one registered model to another as a new model version.
+
+        :param src_model_uri: the model URI of the model version to copy
+        :param dest_name: the name of the registered model to copy the model version to.
+        :return: Single :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                 the cloned model version.
```

Comment:
```suggestion
                 the copied model version.
```

nit: just for consistency.